doc/rst/quick.rst

```diff
@@ -143,81 +133,117 @@ Building the firmware
+   *Note: Some boards have an integrated xTAG.*
 
-Finally, to run the binary on the target, execute a command similar to the following::
+#. Ensure that any required external power supplies are connected.
 
-    xrun path/to/binary.xe
+Once everything is connected, run the binary on the target device using a command like the following::
 
-The device should now present itself as a USB Audio Device on the connected host computer.
-It will continue to operate as a USB Audio Device until the target board is power cycled.
+    xrun ./bin/2AMi10o10xssxxx/app_usb_aud_xk_316_mc_2AMi10o10xssxxx.xe
+
+The device should now appear as a USB Audio Device on the host computer. It will continue to function as such until the board is power cycled.
 
 Writing the application binary to flash
 =======================================
 
-Optionally a binary can be programmed into the boot flash. To do this:
+If desired, the application binary can be programmed into the device’s boot flash. To do so:
 
-    #. Connect the USB Audio board to the host computer.
+#. Connect the USB Audio board to the host computer.
 
-    #. Connect the xTAG to the USB Audio board and connect the it to the host machine on which the application binary is present via a separate USB cable
+#. Connect the xTAG to the USB Audio board, then connect the xTAG to the host computer using a separate USB cable.
+   *Note: Some boards include an integrated xTAG.*
 
-    #. Ensure any required external power jacks are connected
+#. Ensure any required external power supplies are connected.
 
-From a command prompt with the XMOS tools available, run the following command::
+Once everything is connected, flash the binary on the target device using a command like the following::
 
-    xflash path/to/binary.xe
+    xflash ./bin/2AMi10o10xssxxx/app_usb_aud_xk_316_mc_2AMi10o10xssxxx.xe
 
-Once flashed the target device will reboot and execute the binary. Power cycling the target board will cause the device to reboot the flashed binary.
+After flashing, the device will automatically reboot and begin executing the application.
+Subsequent power cycles will cause the device to boot and run the flashed binary.
 
 .. _quick_start_play_audio:
 
-Playing audio
-=============
+Playing/recording audio
+=======================
+
+To play and record audio using the USB Audio board, follow these steps:
 
-    #. Connect the board to any power supply provided (note, some boards will be USB bus powered)
+#. Connect the board to a power supply, if required.
+   *Note: Some boards are powered via USB and do not require an external supply.*
 
-    #. Connect the board to a host with driver support for USB Audio Class using a USB cable
+#. Connect the board to a host system using a USB cable.
+   Ensure the host supports USB Audio Class.
 
-    #. Install the Windows USB Audio 2.0 demonstration driver, if required.
+#. Install the Windows USB Audio 2.0 demonstration driver, if required.
```

Comment:
Are we still recommending this? Is the Windows built-in driver not sufficient. Never quite sure these days.